NOTICE

```diff
@@ -9,3 +9,6 @@ within this component.
 
 Copyright (C) 2012 Raphael Kubo da Costa <rakuco@webkit.org>
 Licensed under the BSD-2 license
+
```

Comment:
Do we need this license entry? What change in PersistentStore is using pxCore?

Reply:
Requested by RDK CMF Compliance Team in #87

Reply:
ok sounds good.